<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="nbPointsLabel">
             <property name="text">
              <string>Number of Points</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="number_points_spinbox">
             <property name="minimum">
              <number>3</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="deltaPhiLabel">
             <property name="text">
              <string>Delta Phi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="delta_phi_textbox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>30</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="start_alignment_button">
             <property name="text">
              <string>Start Aligment</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancel_alignment_button">
             <property name="text">
              <string>Cancel Aligment</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="show_center_checkbox">
           <property name="text">
            <string>Show center</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="show_help_line_checkbox">
           <property name="text">
            <string>Show Help Line</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="aligment_table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="rowCount">
        <number>3</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Î”Ths</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>y</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>z</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
